docs/dependencies.md

```diff
@@ -0,0 +1,59 @@
+### Installing required dependencies for local build
+
+To get requirements installed locally, run:
+
+```bash
+$> make deps
+```
+
+or you need:
+
+- [`luet`](https://github.com/mudler/luet)
+- [`luet-makeiso`](https://github.com/mudler/luet-makeiso)
+- [`squashfs-tools`](https://github.com/plougher/squashfs-tools)
+  - `zypper in squashfs` on SLES or openSUSE
+- [`xorriso`](https://dev.lovelyhq.com/libburnia/web/wiki/Xorriso)
+  - `zypper in xorriso` on SLES or openSUSE
+- `yq` ([version `3.x`](https://github.com/mikefarah/yq/releases/tag/3.4.1)), installed via [packages/toolchain/yq](https://github.com/rancher-sandbox/cOS-toolkit/tree/master/packages/toolchain/yq) (optional)
+- [`jq`](https://stedolan.github.io/jq), installed via [packages/utils/jq](https://github.com/rancher-sandbox/cOS-toolkit/tree/master/packages/utils/jq) (optional)
+
+_Note_: Running `make` deps will install only `luet`, `luet-makeiso`, `yq` and `jq`. `squashfs-tools` and `xorriso` needs to be provided by the OS.
+
+### Manually install dependencies
+
+To install luet locally, you can also run as root:
+```bash
+$> curl https://get.mocaccino.org/luet/get_luet_root.sh | sh
+```
+or either build from source (see [luet](https://github.com/mudler/luet)).
+
+The Luet official repository that are being installed by the script above are:
+- [official Luet repository](https://github.com/Luet-lab/luet-repo)
+- [mocaccino-extra repository](https://github.com/mocaccinoOS/mocaccino-extra) (installable afterwards also with `luet install -y repository/mocaccino-extra-stable`) that contains the `yq` and `jq` versions that are used by the CI. 
+
+
+#### luet-makeiso
+
+Available in the [official Luet
```

Comment:
They are available now in the `cOS` repositories directly, no need to use luet repositories anymore

Reply:
fixed